canonical/microcloud.is
# Add roadshow page

## Done

* added /roadshow
* extracted description to pages so we can customize it

## QA

* check /roadshow content and outgoing links
* check description of other pages

## Issue / Card

WD-6259
